graphql/queries/contact-employment-insurance.graphql

```diff
@@ -0,0 +1,78 @@
+{
+  schListv1ByPath(
+      _path: "/content/dam/decd-endc/content-fragments/sch/lists/contact-us/ei-contact-methods"
```

Comment:
Looks good! But I think we'll need to direct our query towards the page eg. '/sch/pages/contact-us-pages/ei-contacts' and then pull in the contact-methods that are associated with the page fragment as part of a greater query of the page. This way we'll be able to get additional info from the page like its meta data, title, etc.